### PR TITLE
Updated Makefiles as per modified directory structure

### PR DESCRIPTION
**Pull request by karthikamurthy**
Bux Fix: 

1) Build gets stuck at message “Building Avalon lmdb” and
    never comes out of the message On switching between Fabric build
    to Standalone build and then back to any docker builds on same repo

2) Standalone build is failing in the latest master with no module named 'zmq'

### CODE REVIEW

File: enclave_manager/Makefile [hunk 1]
```diff
@@ -38,10 +38,11 @@ install:
 	@echo INSTALLING WHEEL FILE =================
 	pip3 install $(WHEEL_FILE)
 
-clean:
-	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
-	rm -f avalon_enclave_manager/avalon_enclave.py avalon_enclave_manager/avalon_enclave_wrap.cpp
+clean:
```
Comment (danintel) [inline enclave_manager/Makefile:41]: Remove the tab character you added here

File: common/crypto_utils/Makefile [hunk 2]
```diff
@@ -38,10 +38,13 @@ install:
...
 clean:
-	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
 	rm -rf build deps dist *.egg-info
-	rm -f crypto/crypto.py crypto/crypto_wrap.cpp
-	rm -f verify_report/verify_report.py verify_report/verify_report_wrap.cpp
+	rm -f avalon_crypto_utils/crypto/crypto.py
+	rm -f avalon_crypto_utils/crypto/crypto_wrap.cpp
+	rm -f avalon_crypto_utils/verify_report/verify_report.py
+	rm -f avalon_crypto_utils/verify_report/verify_report_wrap.cpp
+	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
```
Comment (manju956) [inline common/crypto_utils/Makefile:46]: Generally wheel file will be copied to deps folder in the directory which has Makefile. We are already deleting build folder before this. Then why do we need this if check?

Comment (karthikamurthy) [reply common/crypto_utils/Makefile:46]: Ya Manju , all wheel file is getting copied to dist folder not deps. So will remove the step of exclusively deleting the wheel file.

File: enclave_manager/Makefile (hunk 1 above)
Comment (karthikamurthy) [reply enclave_manager/Makefile]: Done

File: common/crypto_utils/Makefile (hunk 2 above)
Comment (karthikamurthy) [reply common/crypto_utils/Makefile]: done

File: enclave_manager/Dockerfile [hunk 3]
```diff
@@ -215,6 +215,7 @@ RUN ldconfig \
  && echo "SGX_SSL=/opt/intel/sgxssl" >> /etc/environment
 
 COPY . /project/avalon
+RUN rm -rf /project/avalon/tools/build/_dev
```
Comment (Ram-srini) [inline enclave_manager/Dockerfile:218]: Add comment here why we are removing it. Mention it docker build won't progress if we already has it host.

Comment (karthikamurthy) [reply enclave_manager/Dockerfile:219]: Done

**Pull request merged**